fuzzing/private/fuzz_test.bzl

```diff
@@ -133,6 +133,8 @@ def fuzzing_decoration(
         tags = ["manual"],
     )
 
+# Use += rather than append to support selects.
```

Comment:
I'm not sure I am familiar with this part. Can you elaborate?

Reply:
Good point, I added more context to the comments and linked a relevant Bazel issue.

Reply:
Thanks! Closing this conversation in favor of the new comments.